LICENSE

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [Nurgul Kereikhan]
```

Comment:
- [OPTIONAL] I think it can be better if you update your MIT LICENSE file to look more professional 😎